process/lifecycle.md

```diff
@@ -9,101 +9,68 @@ has_children: true
 * Archived
 
-All projects must meet the Incubation stage requirements and following a successful Graduation Review, progress to Adopted. It is possible that some projects may be approved as Incubation and pass a Graduation Review at the same time to advance directly to Adopted. A project that does not pass a Graduation Review remains at the Incubation stage unless the TAC votes to transition the project to Archive status.  Alternatively, the TAC or the project TSC may vote to exit the lifecycle under specific terms for transfer of trademarks and other key assets.
+Any project to be considered for inclusion by the TAC must complete the [proposal template](proposal_template). More details on bringing a project to the Academy Software Foundation are outlined [here](start_project).
 
 ## Sandbox Stage
 
-Projects being submitted to the ASWF at the sandbox level are intended to be the entry point for early-stage projects. Characteristics for projects at the Sandbox Stage maybe one or more of:
+Projects submitted to the Academy Software Foundation at the Sandbox Stage are intended to be the entry point for early-stage projects. Characteristics for projects at the Sandbox Stage may be one or more of the following:
 
-* Early-stage projects that the ASWF TAC believes warrant experimentation.
-* New projects that are designed to extend one or more TAC projects with functionality or interoperability libraries.
-* Independent projects that fit the ASWF mission/vision and provide the potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
-* Projects commissioned or sanctioned by the ASWF, including initial code for ASWF Working Group collaborations, and "experimental" projects.
-* Any project that realistically intends to join ASWF Incubation in the future and wishes to lay the foundations for that.
+* Early-stage projects that the  TAC believes warrant experimentation.
+* New projects designed to extend one or more TAC projects with functionality or interoperability libraries.
+* Independent projects that fit the Academy Software Foundation mission/vision and provide the potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
+* Projects commissioned or sanctioned by the Academy Software Foundation, including initial code for Academy Software Foundation Working Group collaborations and "experimental" projects.
+* Any project that realistically intends to become an Academy Software Foundation Incubation Stage project and wishes to lay the foundations for that.
 
 ### Requirements
 
 To be accepted at the Sandbox stage, a project must:
 
-* Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC, or the TAC’s designated recipient for contribution proposals.
+* Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC or the TAC’s designated recipient for contribution proposals.
```

Comment:
I thought it was explicitly the TAC's mandate to deal with contribution proposals?

Reply:
It is, I think this is language that has persisted since the first version of this. Removed since yes the TAC deals with the proposals.